packages/contract/src/contract-factory.ts

```diff
@@ -213,6 +357,16 @@ export default class ContractFactory {
     }
   }
 
+  /**
+   * Helper function to get the size of the contract's bytecode. Useful for informing
+   * which deployment method to use.
+   *
+   * @returns The size of the contract's bytecode.
+   */
+  getBytecodeSize(): number {
```

Comment:
IMO there's no need for this method as `this.bytecode.length` / `myContractFactory.bytecode.length` is straightforward.